### ISSUE TITLE
Missing IPV6 Firewall in UI User view

### ISSUE THREAD

**Issue by fermosan** [apache/cloudstack#6575]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.17
~~~

##### CONFIGURATION

IPV6 Isolated Networkers (DUAL STACK)

##### SUMMARY
Under Network -> Guest Networks -> any IPV4+IPV6 (Dualstack) networker the tab IPv6 Firewall is missing for anyone but the Administrator. 

##### STEPS TO REPRODUCE

~~~
Create a new Isolated IPv4 + IPv6 network.
Check network with Administrator and you will see Ipv6 Firewall is present
Check with user that the network belongs to. The Ipv6 Firewall tab is missing.
~~~

##### EXPECTED RESULTS

~~~
Both User and Administrator should be able to view and configure IPV6 Firewall
~~~

##### ACTUAL RESULTS
~~~
Only Administrator can see and configure IPV6 Firewall
~~~

~~~
Admin view:
![image](https://user-images.githubusercontent.com/78906279/181244312-95350d96-1cf0-42ed-9e9b-c514cca35b4f.png)

User: view
![image](https://user-images.githubusercontent.com/78906279/181244464-6865079b-4bd2-42f3-86d1-a582f60ed6b5.png)
~~~

**Comment by weizhouapache** [comment]
> Hi @weizhouapache - is this the expected behavior?

@nvazquez 
this looks like a bug. the owner and domain admin should be able to manage the ipv6 firewalls as well.

**Comment by shwstppr** [comment]
@nvazquez @weizhouapache @fermosan I guess this was a regression with https://github.com/apache/cloudstack/commit/40dabb6ee746495fce9ffc80055337674046b230 and was fixed with https://github.com/apache/cloudstack/commit/d45d257b9a31b5825d87a4f40412c54a3d1ed64c

@fermosan is it possible for you to test this with the lates nightly packages? https://download.cloudstack.org/testing/nightly/latest/

**Comment by shwstppr** [comment]
Apologies @nvazquez @weizhouapache @fermosan. Ignore my previous comment.
I tested and found that the issue is due to IPv6 API access for default roles. I've created a PR to fix it, will test that and update.

**Comment by shwstppr** [comment]
Fixed with [9a264c1](https://github.com/apache/cloudstack/commit/9a264c1369be538d3206170db3ad85c1a7ed5e6a)
